config/initializers/devise.rb

```diff
@@ -5,9 +5,8 @@
   config.case_insensitive_keys = [:email]
   config.strip_whitespace_keys = [:email]
   config.skip_session_storage = [:http_auth]
-  config.stretches = Rails.env.test? ? 1 : 10
+  config.stretches = Rails.env.test? ? 1 : 12
   config.pepper = '0761ef0892f647b6d3c525ee7d98043efadaae4b42d77102b56beba91bd44360d4cfb0ffa713778c281afe99e18c22ed0364330c28c0292fc25251c3f8858331'
```

Comment:
Having the `pepper` here in a public repo makes it useless from a security-standpoint. But it's not _worse_ than not having one (which is the default). And removing/changing it invalidates existing hashed passwords.